scrapers/Staxus.yml

```
@@ -0,0 +1,382 @@
+name: "Staxus"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://staxus.com/trial/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title:
+        selector: //title/text()
+        postProcess:
+          - replace:
+              - regex: 'Staxus Scene - '
+                with: ""
+      Date:
+        selector: //div[@class='video-details'][1]
```

```suggestion
        selector: //div[@class='video-details'][strong='Added:']
```
using the above is safer imo